README.md

```diff
@@ -305,6 +305,95 @@ In particular, note that [vLLM does not yet support function calling](https://gi
 
 </details>
 
+### Install using helm - Run HolmesGPT in your cluster
+
+You can install HolmesGPT in your Kubernetes cluster, and use its rest API to investigate incidents. 
+This works best with the [Robusta-Holmes integration ↗](https://docs.robusta.dev/master/configuration/ai-analysis.html).
+
+In this mode, all the parameters should be passed to the HolmesGPT deployment, using environment variables.
```

Comment:
I think this could be confusing for users. You're implying here that you are giving instructions for installing holmes with the Robusta integration, but actually users who follow the below instructions will end up with Holmes running in the cluster alone

Reply:
fixed